doc/mgr/prometheus.rst

```diff
@@ -41,13 +43,21 @@ configuration parameter. The parameter is a comma or space separated list
 of ``pool[/namespace]`` entries. If the namespace is not specified the
 statistics are collected for all namespaces in the pool.
 
+Example to enable the images ``image1``, ``image2`` and ``imageN``::
+
+  ceph config set mgr mgr/prometheus/rbd_stats_pools "image1,image2,imageN"
+
 The module makes the list of all available images scanning the specified
 pools and namespaces and refreshes it periodically. The period is
 configurable via the ``mgr/prometheus/rbd_stats_pools_refresh_interval``
 parameter (in sec) and is 300 sec (5 minutes) by default. The module will
 force refresh earlier if it detects statistics from a previously unknown
 RBD image.
 
+Example to turn up the sync interval to 10 minutes::
+
+  ceph config set mgr mgr/prometheus/rbd_stats_pools_refresh_interval 600
```

Comment:
Did you try that command? I just did it, and I may be wrong, but it's broken (not the command itself, the prometheus exported code that handles that option):
```py
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/cherrypy/_cprequest.py", line 638, in respond
    self._do_respond(path_info)
  File "/usr/lib/python3.6/site-packages/cherrypy/_cprequest.py", line 697, in _do_respond
    response.body = self.handler()
  File "/usr/lib/python3.6/site-packages/cherrypy/lib/encoding.py", line 219, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/cherrypy/_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/ceph/src/pybind/mgr/prometheus/module.py", line 1058, in metrics
    return self._metrics(instance)
  File "/ceph/src/pybind/mgr/prometheus/module.py", line 1073, in _metrics
    instance.collect_cache = instance.collect()
  File "/ceph/src/pybind/mgr/prometheus/module.py", line 976, in collect
    self.get_rbd_stats()
  File "/ceph/src/pybind/mgr/prometheus/module.py", line 737, in get_rbd_stats
    'rbd_stats_pools_refresh_interval', 300)
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```
To begin with, the default is currently hard-coded to `300` in the `prometheus/module.py`, but not using the proper Ceph config option `default` field.

Since you're dealing with this, I'd encourage you to try and fix this issue in the corresponding prometheus `MODULE_OPTIONS` section:
```diff
    MODULE_OPTIONS = [
        {'name': 'server_addr'},
        {'name': 'server_port'},
        {'name': 'scrape_interval'},
        {'name': 'rbd_stats_pools'},
-        {'name': 'rbd_stats_pools_refresh_interval'},
+        {'name': 'rbd_stats_pools_refresh_interval', type: , type: 'int', default: 300},
    ]
```

Additionally, that would allow, for example:
* ceph-dashboard to properly detect and display the default value for that setting:
![image](https://user-images.githubusercontent.com/37327689/76882718-5ab07000-687b-11ea-848f-ed56d3af1695.png)

* `ceph config *` to work with a known default.

Reply:
I just copy and pasted my command. Executed it against my ses6 testcluster without problems. `ceph config dump` reported the correct value. Also when I try to apply your patch my Python Syntax checker in CLion (so Pycharm) is not happy. I will (as written underneath) work on the Ceph Dashboard after this is merged.

Reply:
> I just copy and pasted my command. Executed it against my ses6 testcluster without problems. `ceph config dump` reported the correct value.

Yup, but did you check your ceph-mgr logs (you might see this exception there)? Or if the prometheus exporter is actually exporting data?

> Also when I try to apply your patch my Python Syntax checker in CLion (so Pycharm) is not happy. I will (as written underneath) work on the Ceph Dashboard after this is merged.

Sorry, my bad. Too much JS recently. Already fixed it in place (please check https://github.com/ceph/ceph/pull/33991#discussion_r393843336 again).

Reply:
Nope indeed that I did not do. I will check.

No worries. Now syntax seems to be fine! Although I need to switch from our sesdev to the ceph scripts for testing if it actually worked.

Reply:
Yes you are right... Setting that option results in a Python Stacktrace when the exporter is called...

Reply:
I was able to use your suggestion and test it using `make vstart`. I consider this resolved